bokeh/models/graphs.py

```diff
@@ -76,7 +74,8 @@ class StaticLayoutProvider(LayoutProvider):
 
     '''
 
-    graph_layout = Dict(Either(String, Int), Seq(Any), default={}, help="""
+    # TODO: Seq(Any).length == 2
+    graph_layout = Dict(Int, Seq(Any), default={}, help="""
```

Comment:
This is because originally this mapped to `{[key: string]: Arrayable<any>}` in JS (well, technically `string | number`, but numbers are treated as strings in this context anyway). So, `String` was the correct type and `Int` was a convenience. However, `Int` is the actual correct type and now this maps to `Map<number, Arrayable<any>>` in JS.

Reply:
`graphs.py` was doing pointless `.astype(str)`. Perhaps this preceded adding `Int` to the keys' type.